contents/blog/hiring-myths-and-truths.md

```diff
@@ -0,0 +1,75 @@
+---
+date: 2022-01-20
+title: Basic myths and truths about hiring that startups keep getting wrong
```

Comment:
Possible headlines:

- Lies people believe about startup hiring and how to do it right
- Hiring at Startups: The lies people believe and the stuff that works
- Common myths about startup hiring (and how to do it right)
- What I've learned from 10 years of hiring for startups

Reply:
How about 'Myths and truths: what I've learned from 10 years of startup hiring' (I thought it would be good to get myths in the title somehow as it seems like a decent hook?). Editor's choice though - I like the last one best.